### ISSUE TITLE
Deploy a release to maven.scijava.org

### ISSUE THREAD

**Issue by ctrueden** [trackmate-sc/TrackMate-StarDist#5]
This component is not currently available from maven.scijava.org, because one of its dependencies, `de.csbdresden:StarDist_`, is not deployed there. We need to make `de.csbdresden:StarDist_` available from https://maven.scijava.org/content/groups/public and then cut a release of TrackMate-StarDist.

**Comment by tinevez** [comment]
Shall we defer to the StarDist maintainers instead?

**Comment by ctrueden** [comment]
@tinevez Yes, ideally https://github.com/stardist/stardist-imagej would have SciJava GitHub Actions config added, and deployment secrets added to the stardist org. CC @uschmidt83

In the meantime, I have deployed [`de.csbdresden:StarDist_:0.3.0-scijava`](https://maven.scijava.org/#nexus-search;gav~de.csbdresden~StarDist_~0.3.0-scijava~~), with lightly modified POM from the 0.3.0 tag, to maven.scijava.org, along with its needed dependency [Clipper](https://maven.scijava.org/#nexus-search;gav~de.lighti~Clipper~~~), and then [released TrackMate-StarDist 1.1.4](https://maven.scijava.org/#nexus-search;gav~sc.fiji~TrackMate-StarDist~1.1.4~~) with fixed dependencies (0ad838a2035d0e923159ce34746bcecc136d504c).

<details><summary>Here is the patch I did to stardist-imagej's pom.xml</summary>

```diff
diff --git a/pom.xml b/pom.xml
index 4e4cfb4..299d44a 100644
--- a/pom.xml
+++ b/pom.xml
@@ -5,13 +5,13 @@
 	<parent>
 		<groupId>org.scijava</groupId>
 		<artifactId>pom-scijava</artifactId>
-		<version>28.0.0</version>
+		<version>33.2.0</version>
 		<relativePath />
 	</parent>
 
 	<groupId>de.csbdresden</groupId>
 	<artifactId>StarDist_</artifactId>
-	<version>0.3.0</version>
+	<version>0.3.0-scijava</version>
 
 	<name>StarDist</name>
 	<description>StarDist - Object Detection with Star-convex Shapes</description>
@@ -89,7 +89,7 @@
 	<mailingLists>
 		<mailingList>
 			<name>Image.sc Forum</name>
-			<archive>https://forum.image.sc/</archive>
+			<archive>https://forum.image.sc/tag/stardist</archive>
 		</mailingList>
 	</mailingLists>
 
@@ -112,12 +112,15 @@
 		<main-class>de.csbdresden.stardist.StarDist2D</main-class>
 		<license.licenseName>bsd_3</license.licenseName>
 		<license.copyrightOwners>Uwe Schmidt, Martin Weigert</license.copyrightOwners>
+
+		<Clipper.version>6.4.2</Clipper.version>
+		<csbdeep.version>0.6.0</csbdeep.version>
 	</properties>
 
 	<repositories>
 		<repository>
-			<id>imagej.public</id>
-			<url>https://maven.imagej.net/content/groups/public</url>
+			<id>scijava.public</id>
+			<url>https://maven.scijava.org/content/groups/public</url>
 		</repository>
 	</repositories>
 
@@ -135,19 +138,16 @@
 			<groupId>net.imagej</groupId>
 			<artifactId>imagej-legacy</artifactId>
 		</dependency>
-		<!-- Clipper -->
 		<dependency>
 			<groupId>de.lighti</groupId>
 			<artifactId>Clipper</artifactId>
-			<version>6.4.2</version>
-			<scope>system</scope>
-			<systemPath>${basedir}/lib/Clipper-6.4.2.jar</systemPath>
+			<version>${Clipper.version}</version>
 		</dependency>
 		<!-- CSBDeep -->
 		<dependency>
 			<groupId>de.csbdresden</groupId>
 			<artifactId>csbdeep</artifactId>
-			<version>0.3.5-SNAPSHOT</version>
+			<version>${csbdeep.version}</version>
 		</dependency>
 		<!-- Utils -->
 		<dependency>
```

</details>

So this issue is resolved. The only future challenge will be if stardist-imagej makes another release in the future without using the SciJava release process (and is thus not available on maven.scijava.org), then updating this component to use that version will require another patch+deploy like the above one.
